client/components/chart/x-axis.jsx

```diff
@@ -3,74 +3,62 @@
 /**
  * External dependencies
  */
-
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { PureComponent } from 'react';
+import { numberFormat } from 'i18n-calypso';
 import { throttle } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import Label from './label';
 
-export default class extends React.Component {
+export default class ModuleChartXAxis extends PureComponent {
 	static displayName = 'ModuleChartXAxis';
 
 	static propTypes = {
 		labelWidth: PropTypes.number.isRequired,
 		data: PropTypes.array.isRequired,
 	};
 
+	constructor( props ) {
```

Comment:
I think you can skip the constructor here and just add the class variables like so:

```
export default class ModuleChartXAxis extends PureComponent {
  axisRef = React.createRef();
  axisSpacerRef = React.createRef();
```

Reply:
Ah, right! Good call. I'll ship it with this change.